### PR TITLE
Tarot rejuvenation

### PR DESCRIPTION
**Pull request by kaylynn234**
This PR makes tarot a little bit nicer.
Changes include:
- Creation of the coveted exceptions.py (which will no doubt be expanded upon later)
- Allowing ``>tarot card`` to be used with the name of a card or the card picking UX. This uses some fancy fuzzy matching so you can afford to make small mistakes in your spelling! Mobile users rejoice.
- Some shuffling of functionality around so that it's tidier.
- Some bug crunching.

### CODE REVIEW

File: crimsobot/cogs/mystery.py [hunk 1]
```diff
@@ -4,16 +4,106 @@
...
+
+            except ValueError:
+                return False
+
+        # wait for user to spcify suit
+        try:
+            msg = await self.bot.wait_for('message', check=suit_check, timeout=45)
+        except asyncio.TimeoutError:
+            await prompt_suit.delete()
+            raise NoMatchingTarotCard
```
Comment (williammck) [inline crimsobot/cogs/mystery.py:50]: why not just `raise` so we re-raise and then catch `asyncio.TimeoutError` in `card()`?
same applies for prompt 2.

File: crimsobot/cogs/mystery.py [hunk 2]
```diff
@@ -4,16 +4,106 @@
...
+
+        await prompt_suit.delete()
+
+        if msg is None:
+            raise NoMatchingTarotCard
```
Comment (williammck) [inline crimsobot/cogs/mystery.py:55]: do we still need this? why would it be `None`?
same applies for prompt 2.

Comment (kaylynn234) [reply crimsobot/cogs/mystery.py:55]: Not sure. This is crimso code so I left it here but it seems like it's worth removing. Will update.

File: crimsobot/cogs/mystery.py (hunk 1 above)
Comment (kaylynn234) [reply crimsobot/cogs/mystery.py:50]: Good point. Will update.

**Pull request merged**